<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExcludedLabelsDialog</class>
 <widget class="QWidget" name="ExcludedLabelsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QToolButton" name="addButton">
     <property name="toolTip">
      <string>Add this label to the list of excluded labels</string>
     </property>
     <property name="icon">
      <iconset theme="list-add"/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="KLineEdit" name="addLabelLine">
     <property name="trapEnterKeyEvent" stdset="0">
      <bool>true</bool>
     </property>
     <property name="clickMessage">
      <string>Add Label to Exclude</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="textLabel">
     <property name="text">
      <string>Selected labels won't be touched by statistics synchronization</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
